models/user.js

```diff
@@ -0,0 +1,37 @@
+const bcrypt = require("bcryptjs");
+module.exports = (sequelize, DataTypes) => {
```

Comment:
small nit pick, but you generally want a newline between your imports and the export